README.md

```diff
@@ -32,7 +32,7 @@ The following table reflects the head of this branch.
 | Feature       | Status  | Default | Description                                                                                   |
 | ------------- | ------- | ------- | --------------------------------------------------------------------------------------------- |
 | CSINode*      | Beta    | On      | external-attacher uses the CSINode object to get the driver's node name instead of the Node annotation. |
-| CSIMigration* | Alpha   | On      | [Migrating in-tree volume plugins to CSI](https://kubernetes.io/docs/concepts/storage/volumes/#csi-migration). |
+| CSIMigration* | Beta    | On      | [Migrating in-tree volume plugins to CSI](https://kubernetes.io/docs/concepts/storage/volumes/#csi-migration). |
```

Comment:
CSINode is GA now, but we still use beta object for backwards compatibility. How do you want to note it? Or can we remove it altogether? Our min version is 1.15 which means that it should not be falling back on node annotation anymore

Reply:
Have we ever removed `storage.k8s.io/v1beta1`? IMO we've always added new objects there (+v1alpha1) and we do not deprecate / remove it. We can keep using it and move to v1 in the next release, bumping just min. k8s version.

We should not use the node annotation, I filed https://github.com/kubernetes-csi/external-attacher/issues/203.